src/lib/konnectors.js

```diff
@@ -6,8 +6,15 @@ export const KONNECTORS_DOCTYPE = 'io.cozy.konnectors'
 export const KONNECTORS_RESULT_DOCTYPE = 'io.cozy.konnectors.result'
 
 export const KONNECTOR_STATE = {
-  READY: 'ready',
-  ERRORED: 'errored'
```

Comment:
We don't have errored state anymore? How do you catch errors so?

Reply:
Quoting IRC:
```
[15:43:27]  <gregory>	Et 'available' signifie quoi exactement ?
[15:44:32]  <gregory>	Ne vous manquerait-il pas toutefois le 'errored' ?
[15:48:22]  <pierrot>	gregory: nope, ça n'existe plus
[15:48:49]  <pierrot>	si une installe échoue, elle ne va pas laisser un document derrière elle en état "errored"
[15:49:35]  <pierrot>	Available c'est un "relica" comme on dit, ce n'est pas utilisé
[15:51:13]  <gregory>	Ok, je le vire de collect pour éviter les incompréhensions, merci !
[15:51:28]  <gregory>	Je vire aussi erreored 👍
[15:51:41]  <gregory>	Du coup si une install échoue il y a zéro documents ?
[15:52:12]  <pierrot>	yep
[15:52:35]  <pierrot>	et si une update échoue, le document est conservé
[15:52:48]  <pierrot>	les erreurs sont remontées dans le realtime par contre
[16:14:57]  <gregory>	ok :)
```

Reply:
Well, nice so 👍